.github/CODEOWNERS

```
@@ -93,6 +93,7 @@
 /rum_javascript/                         @DataDog/rum-app
 /k6/                                     @ppcano support@k6.io
 /kernelcare/                             @grubberr schvaliuk@cloudlinux.com
+/keep/                                   @talboren tal@keephq.dev @DataDog/documentation
```

`documentation` is defined for reviews in Line 4 above  🙌🏼

```suggestion
/keep/                                   @talboren tal@keephq.dev @DataDog/ecosystems-review
```